endpoints/remotehost/remotehost

```diff
@@ -394,8 +473,4 @@ init_common_dirs
 remotehost_req_check
 base_req_check
 launch_osruntime
-# process_prebench_roadblocks
-# process_bench_roadblocks remotehost
-# process_postbench_roadblocks
 process_roadblocks remotehost
-endpoint_remotehost_cleanup
```

Comment:
Removing this line seems broken.  I just added this back so this might be caused by an issue with rebasing your tree (https://github.com/perftool-incubator/rickshaw/commit/eee8d3b58ff59fb144509ae5b3582b27847c05a9).

Reply:
Removing it it not broken.  endpoint_${endpoint}_cleanup is called in process_roadblocks()

Reply:
Ah, I guessed I missed that.  Carry on.